src/app/copilot/components/copilot-panel/copilot-panel.component.scss

```
@@ -0,0 +1,1561 @@
+    &__content {
+      display: flex;
+      flex-direction: column;
+      max-width: 72%;
+      min-width: 0;
+    }
+
+    &__bubble {
+      padding: ($u * 2) ($u * 2 + 4px);
+      font-size: 14.5px;
+      line-height: 1.7;
+      word-wrap: break-word;
```

Stylelint (17.12.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)